app/v1/audit/AuditEvents.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.audit
+
+import v1.models.audit.{AuditDetail, AuditEvent, AuditResponse}
+import v1.models.auth.UserDetails
+
+object AuditEvents {
+
+  def auditReturns(nino: String, correlationId: String, userDetails: UserDetails, auditResponse: AuditResponse): AuditEvent = {
```

Comment:
shall we make a common method with params **AuditDetail**, **AuditType** and **transactionName**

Reply:
Do we really need this class?? or we can remove this once we implement the controllers